scripts/ci-build.sh

```diff
@@ -261,16 +261,22 @@ define_with_default KEEP_TMP false
 
 stop_if_failure
 
-git_gdp="https://github.com/GENIVI/genivi-dev-platform"
```

Comment:
Nit pick: This section seems unrelated to adding salvator-x, i.e. should it be stand alone "clean-up".